plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-homepage.tsx

```diff
@@ -223,10 +223,8 @@ export const SidebarNavigationScreenHomepage = ( {
 
 	const isNetworkOffline = useNetworkStatus();
 	const isPTKPatternsAPIAvailable = context.isPTKPatternsAPIAvailable;
-	const trackingAllowed = useSelect( ( sel ) =>
-		sel( OPTIONS_STORE_NAME ).getOption( 'woocommerce_allow_tracking' )
-	);
-	const isTrackingDisallowed = trackingAllowed === 'no' || ! trackingAllowed;
+
+	const isTrackingAllowed = useTrackingAllowed();
```

Comment:
It's really a minor complaint/nitpick, but I don't think you should trick the consumer of the API into thinking this is a hook when it's not. E.g. you'd be inclined to think it follows or must implement the rules of hooks when it's just a plain function .

Reply:
💯 agree! Thanks for pointing this out! Fixed with [00f9f8e](https://github.com/woocommerce/woocommerce/pull/48767/commits/00f9f8e426f743d5b2d44295ba385c85ef892fd0)